src/qt/optionsmodel.h

```diff
@@ -123,6 +126,7 @@ class OptionsModel : public QAbstractListModel
 Q_SIGNALS:
     void displayUnitChanged(int unit);
     void coinControlFeaturesChanged(bool);
+    void SubFeeFromAmountChanged(bool);
```

Comment:
Why is this Q_SIGNAL needed if it isn't `connect()`ed anywhere?

Reply:
Addressed the suggested changes in the recent commit.